pkg/models/oidc_clients.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2025 Siemens AG
+// SPDX-FileContributor: Dearsh Oberoi <dearsh.oberoi@siemens.com>
+//
+// SPDX-License-Identifier: GPL-2.0-only
+
+package models
+
+// OidcClient struct for Database Layer.
+type OidcClient struct {
+	Id       int64  `gorm:"primary_key;column:id"`
+	ClientId string `gorm:"column:client_id"`
+	UserId   int64
+	User     User `gorm:"foreignKey:UserId;references:Id"`
+}
```

Comment:
A suggestion, it would be good to capture the date client was added (`now()`) for audit purpose as we are not storing the audit logs for client (and it does not make sense to store the audit logs).